src/logger.py

```diff
@@ -129,6 +129,14 @@ def log_error(text, state, level=0):
         log(text, level, state, COLORS['FAIL'])
 
 
+def log_init(text, state, level=2):
```

Comment:
Merge this function with `log_cfg`, setting the default verbosity to 2 by default.